readme redesign (#435)
LargeModGames/spotatui: README.md

<h1 align="center">

>   <img src=".github/spotatui-logo.png" alt="spotatui logo"><br>

Reviewer: The PNG is 1024x1024 and GitHub only caps images at container width, so this renders as a near-full-width square pushing the badges and demo GIF below the fold. Pinning a width fixes it.
```suggestion
  <img src=".github/spotatui-logo.png" alt="spotatui logo" width="200"><br>
```

[![Upstream Contributors](https://img.shields.io/badge/upstream_contributors-94-orange.svg?style=flat-square)](#upstream-contributors-spotify-tui)

![Demo](.github/demo.gif)

## Table of Contents
<!-- START doctoc generated TOC please keep comment here to allow auto update -->

> <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->

Reviewer: This heading is doctoc's own generated title (toc.yml sets `TOC_TITLE: "## Table of Contents"`), so it needs to stay inside the markers. Outside them, the workflow will insert a second copy on the next push to main.
```suggestion
<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
## Table of Contents
```

- [Features](#features)
- [Installation](#installation)
- [Quickstart](#quickstart)
  - [Adding Spotify later](#adding-spotify-later)
- [Music Sources](#music-sources)
  - [Local Files](#local-files)
  - [Subsonic / Navidrome](#subsonic--navidrome)
  - [Internet Radio](#internet-radio)
  - [YouTube](#youtube)
- [AI DJ](#ai-dj)
- [Native Streaming](#native-streaming)
- [Configuration](#configuration)
  - [Discord Rich Presence](#discord-rich-presence)
  - [Anonymous Song Counter](#anonymous-song-counter)
  - [GitHub Profile Widget](#github-profile-widget)
- [Plugins](#plugins)
- [Performance](#performance)
- [Playback Requirements](#playback-requirements)
  - [Deprecated Spotify API Features](#deprecated-spotify-api-features)
- [Using with spotifyd](#using-with-spotifyd)
- [Migrating from spotify-tui](#migrating-from-spotify-tui)
- [Song History](#song-history)
- [Star History](#star-history)
- [Libraries used](#libraries-used)
- [Development](#development)

>   - [If you use NixOS or Nix in general:](#if-you-use-nixos-or-nix-in-general)

Reviewer: This TOC is stale: this entry points at `#if-you-use-nixos-or-nix-in-general` but the heading is now `### Nix`, so the link is dead, and the new About section is missing from the list entirely. Easiest is to re-run `npx doctoc README.md` once the section layout settles.

  - [Windows Subsystem for Linux](#windows-subsystem-for-linux)
- [Help Wanted](#help-wanted)
- [Maintainer](#maintainer)
- [spotatui Contributors](#spotatui-contributors)
- [Upstream Contributors (spotify-tui)](#upstream-contributors-spotify-tui)
- [Roadmap](#roadmap)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## About
<!--
A community-maintained, actively developed fork of [spotify-tui](https://github.com/Rigellute/spotify-tui).
-->

> Originally a fork of [spotify-tui](https://github.com/Rigellute/spotify-tui), but since then the project has diverged in multiple fronts. 

Reviewer: No need to keep the old tagline in a comment, git history has it. The rendered text also drops the "community-maintained, actively developed" signal, which is the main thing someone comparing forks looks for, and "in multiple fronts" should be "on multiple fronts".
```suggestion
A community-maintained, actively developed fork of [spotify-tui](https://github.com/Rigellute/spotify-tui) that has since diverged on multiple fronts.
```

| Key differences | spotify-tui | spotatui |
|---|---|---|

> | Spotify backend | spotifyd | native client |

Reviewer: This misstates spotify-tui: it is a Web API / Spotify Connect controller that works with any Connect device, spotifyd is just one option (this README's own Playback Requirements section says the same). Suggest:
```suggestion
| Spotify backend | external Connect device (spotifyd, official client, ...) | built-in native client |
```

- **Full CLI.** Most of what the UI does is scriptable — playback, search, playlists, shell completions. Run `spotatui --help`.
- **Lightweight.** ~78 MB RAM while streaming, versus a full Electron client. See [Performance](#performance).

> ## Installation 

Reviewer: Trailing space (markdownlint MD009). Same on line 148's `#your other settings `, plus doubled blank lines before "Using with spotifyd".
```suggestion
## Installation
```

}
```

The flake will take care of all the dependencies for you, except for setting the audio backend.
If you use alsa:
   ```nix
    hardware.alsa.enable = true;
   ```
Or if you use pipewire:
   ```nix
    services.pipewire = {
        enable = true;
        alsa.enable = true;
        #your other settings 
    };
   ```

```nix
# NixOS with cargo
nix develop
cargo install --path .

> ```

Reviewer: These commands fail where the reader is: every other block in Installation is copy-paste-able from anywhere, but `nix develop` / `cargo install --path .` need a checkout of the repo (and flakes enabled). The from-source path already lives in the Development section and docs/installation.md, so I'd drop this block. If you want to keep it, add the `git clone` + `cd` steps and use a `bash` fence, these are shell commands, not Nix.
```suggestion
```

1. Clone or fork this repo and `cd` to it
1. And then `cargo run`

### Nix
   ```bash
    nix develop
   ```

>  

Reviewer: This duplicates step 4 right above (`nix develop github:LargeModGames/spotatui`), which makes five places that now give diverging Nix instructions. Suggest dropping this subsection and keeping the step-4 line as the one dev-shell reference.
```suggestion
```
